gpii/node_modules/matchMakerFramework/src/MatchMakerCapabilities.json

```diff
@@ -1,49 +1,38 @@
 [
     {
-        "matchMakerId": "flat",
-        "conditions": {
-            "supported": false
+        "matchMakerId": "Statistical1",
```

Comment:
Hi Kasper.
Oh i thought we would be using a separate pull for demo 3, i will change the file so that it includes the smm prime and rbmm.